Tests/SwiftDocCTests/Rendering/DefaultAvailabilityTests.swift

```diff
@@ -577,6 +577,208 @@ class DefaultAvailabilityTests: XCTestCase {
+            """
+        }
+        func setupContext(
+            inheritDefaultAvailability: String = "",
+            defaultAvailability: String? = nil
+        ) throws -> DocumentationContext {
+            // Create an empty bundle
+            let targetURL = try createTemporaryDirectory(named: "test.docc")
+            // Create symbol graph
+            let symbolGraphURL = targetURL.appendingPathComponent("MyModule.symbols.json")
+            try symbolGraphString.write(to: symbolGraphURL, atomically: true, encoding: .utf8)
+            // Create info plist
+            let infoPlistURL = targetURL.appendingPathComponent("Info.plist")
+            let infoPlist = makeInfoPlist(inheritDefaultAvailability: inheritDefaultAvailability, defaultAvailability: defaultAvailability)
+            try infoPlist.write(to: infoPlistURL, atomically: true, encoding: .utf8)
+            // Load the bundle & reference resolve symbol graph docs
+            let (_, _, context) = try loadBundle(from: targetURL)
+            return context
+        }
+        
+        let symbols = """
+        {
+            "kind": {
+                "displayName" : "Instance Property",
+                "identifier" : "swift.property"
+            },
+            "identifier": {
+                "precise": "c:@F@SymbolWithAvailability",
+                "interfaceLanguage": "swift"
+            },
+            "pathComponents": [
+                "Foo"
+            ],
+            "names": {
+                "title": "Foo",
+            },
+            "accessLevel": "public",
+            "availability" : [
+                {
+                   "domain" : "ios",
+                   "introduced" : {
+                        "major" : 10,
+                        "minor" : 0
+                   }
+                }
+            ]
+        },
+        {
+            "kind": {
+                "displayName" : "Instance Property",
+                "identifier" : "swift.property"
+            },
+            "identifier": {
+                "precise": "c:@F@SymbolWithoutAvailability",
+                "interfaceLanguage": "swift"
+            },
+            "pathComponents": [
+                "Foo"
+            ],
+            "names": {
+                "title": "Bar",
+            },
+            "accessLevel": "public"
+        }
+        """
+        let symbolGraphString = makeSymbolGraphString(
+            moduleName: "MyModule",
+            symbols: symbols,
+            platform: """
+            "operatingSystem" : {
+               "minimumVersion" : {
+                 "major" : 10,
+                 "minor" : 0
+               },
+               "name" : "ios"
+             }
+            """
+        )
+        
+        // Don't use default availability version for symbols.
         
+        var context = try setupContext(inheritDefaultAvailability: """
+        <key>CDInheritDefaultAvailability</key>
+        <string>platformOnly</string>
```

Comment:
Question: how likely do you think it is that we'll want to support more values here?

If we want this to behave like a feature flag then we could make it a boolean value in the property list and move the specifics of the what it does into the name. For example, as a "negative" value:
```xml
<key>CDDisplayInheritDefaultIntroducedVersions</key>
<false/>
```
or as a "positive" value
```
```xml
<key>CDDisplayAvailabilityWithoutInheritVersions</key>
<true/>
```

(I'm certain that there are better names that are both positive and negative for this)

I'm asking because I have some past experience of naming default values as a behavior that people somehow find and and then people start redundantly adding it because the medium can't communicate that it's already the default and that they get the same behavior by doing nothing.